notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1206 @@
+{
```

Comment:
<p>Please add the following underneath the conditional:</p><p> </p><p>"The testing suite does not currently support testing on Dataproc clusters, so the testing environment is setup to replicate Dataproc via the following steps."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>